fastlane/metadata/en-US/promotional_text.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
Flyve MDM is a Mobile device management software that enables you to secure and manage all the mobile devices of your business or family via a web-based console.